oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/validation/JwtValidators.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.jwt.validation;
+
+import org.springframework.security.oauth2.core.valiation.OAuth2TokenValidator;
+import org.springframework.security.oauth2.jwt.Jwt;
+
+/**
+ * @author Josh Cummings
+ * @since 5.1
+ */
+public final class JwtValidators {
```

Comment:
I think we can remove this class and just use `new JwtTimestampValidator()` wherever this class is used.

Reply:
So, check out this PR, too, and let me know what you think:

https://github.com/spring-projects/spring-security/pull/5628/files#diff-853f64a1c30307aeb5132328ffcc75abR60

It's not apparent in this PR alone, but having this class heads off the eventual problem of needing to reconsolidate places that are needing to specify the "default set" of validators. Personally, I'm concerned about two things:

1. If we don't consolidate now, we forget to change all the places in the future.
2. This is convenient for users who are building their own decoders and want to include the standard validator set.

I'm okay with removing it, but what do you think in the context of those two concerns?

Reply:
I see your point after looking at the other PR. Ok let's leave this in for now and see what @rwinch thinks.

Have you also considered adding something like

`JwtValidators.composeValidators(OAuth2TokenValidator<Jwt>... jwtValidators)`

This would return a `DelegatingOAuth2TokenValidator` composed with `defaultJwtValidator()` and the passed in `jwtValidators`?

Btw, I don't like the method name but just putting this idea out there for you to consider.

Reply:
Hmm, I like that. It reminds me a little bit of `PasswordEncoderFactories#creatingDelegatingPasswordEncoder`, which adds the list of standard password encoders.

Maybe `createDelegatingValidator(OAuth2TokenValidator<Jwt>... jwtValidators)`.

Reply:
@jgrandja it doesn't show in this comment thread, but I did add this, feel free to check it out. I think the method is much more useful now. Thanks for the idea.

Reply:
I like it!